### DISCUSSION TITLE
Streamlink Disconnecting on High network loads on 2.1.2/2.2.0?

### DISCUSSION THREAD

**Discussion by chorong761** [streamlink/streamlink#3858] [Q&A]
After updating from 2.0.0 to 2.1.2 and 2.2.0, I've noticed that streamlink would get disconnected easily when there is a high network load. With 2.0.0, the disconnection rate is much lower.

I've tested capturing 10 copies of a same afreeca stream comparing 2.1.2/2.2.0 with 2.0.0 multiple times. The new versions would disconnect/stop capturing a few of the 10 streams tested. With 2.0.0, it would be stable.

Anyone encountering similar problems? I'm forced to update to newer versions due to youtube issues.

**Comment by ferdiko** [comment]
I might have a similar issue. 10 streams and the same 3 of them frequently get a read timeout. What did you end up doing?

**Reply by chorong761** [reply]
> I might have a similar issue. 10 streams and the same 3 of them frequently get a read timeout. What did you end up doing?

I'm using 2.0.0 for the affected streams/sites and the newer version for other sites.
